app/controllers/peeps_controller.rb

```diff
@@ -0,0 +1,15 @@
+class PeepsController < Sinatra::Base
```

Comment:
very interesting, Did you make separate classes for these controllers to adhere to the single responsibility method?

Reply:
Yes exactly!